config/log.go

```diff
@@ -64,6 +65,16 @@ func logProcessor(ctx context.Context, processor LogRecordProcessor) (sdklog.Pro
 }
 
 func logExporter(ctx context.Context, exporter LogRecordExporter) (sdklog.Exporter, error) {
+	if exporter.Console != nil && exporter.OTLP != nil {
+		return nil, errors.New("must not specify multiple exporters")
+	}
+
+	if exporter.Console != nil {
+		return stdoutlog.New(
+			stdoutlog.WithPrettyPrint(),
+		)
+	}
+
```

Comment:
Would be good to add a test for the config change, see https://github.com/open-telemetry/opentelemetry-go-contrib/blob/af75717ac4fb3ba13eaea83b88301723122060cf/config/metric_test.go#L446-L455 for an example

Reply:
Done